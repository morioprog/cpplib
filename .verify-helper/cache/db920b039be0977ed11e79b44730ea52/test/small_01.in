2 20
567927767237 744951463146
0 0 1 193501509873
1 0 1 -682661780049
1 0 1 -790475441901
0 0 2 -755836712066
3 0 1
3 0 1
1 0 2 -893052963231
1 0 1 -135794566542
3 0 1
0 1 2 -451789032885
2 0 2 732269326746
2 1 2 270031476994
3 1 2
2 0 2 -1017434892004
1 0 1 394723063339
2 1 2 509191244995
1 1 2 -941902812323
0 0 1 803055716438
1 1 2 -118925996149
3 0 1
