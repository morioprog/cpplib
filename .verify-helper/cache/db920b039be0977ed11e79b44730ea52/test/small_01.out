-755836712066
-755836712066
-135794566542
246464091674
394723063339
